jenkins/vm-reset

```diff
@@ -61,18 +60,26 @@ validate_args() {
 
 wait_till() {
```

Comment:
please add some documentation on how this function is supposed to be used and what the arguments mean.
May be we can still simplify the use of it but let's complete this patch. If the required functionality is working its okay. 
Just add some documentation for this function.

Reply:
@mssawant  added documentation for wait_till() function.